src/NPCManager.cpp

```diff
@@ -15,7 +15,9 @@ std::vector<WarpLocation> NPCManager::RespawnPoints;
 
 void NPCManager::init() {
     // load NPCs from NPCs.json into our NPC manager
-
+    // Temporary fix, IDs will be pulled from json later
+    int i;
```

Comment:
i = 0

Reply:
Huh, i thought calling int i; would start it from 0, nevertheless done.